test/jset.setup.js

```diff
@@ -0,0 +1,12 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+jest.setTimeout(9000)
```

Comment:
this file is not named properly, typo: `jset` should be `jest`

this file cannot run stand-alone, it must be referred in a jest config file. For example:   https://github.com/adobe/aio-lib-campaign-standard/blob/3ddcbe1c1815b2256cd279d72bbedc3b6eca6499/package.json#L13

Reply:
moved timeout setting to beforeAll function in e2e.js. deleting this file.